README.md

```diff
@@ -45,6 +45,17 @@ Phone (one-tap):
 Phone (international):
 - https://VMware.zoom.us/u/asnmqyfmb
 
+#### Agenda
+
+A typical meeting might have the following agenda:
+
+10:00 - 10:15 - introductions, release updates, housekeeping
```

Comment:
Perhaps working group isn’t the correct forum, but I’d benefit from a segment explicitly about **Priority RFCs/Issues**.

There’s loads to work on, but our community isn’t large enough where all initiatives can be executed at the same time. And not all issues have the same level of impact. As a contributor, I’d like to know which initiatives (_and what topics_) have the highest impact. What _I_ want to work on isn't necessarily what the steering committee has agreed upon.

My best answer for this is our [ROADMAP](https://github.com/buildpacks/community/blob/main/ROADMAP.md) (_which is great_) but I only see 2 RFCs listed there. Participate in the community enough you can get a sense of "what’s not being listed", but writing the RFC requires a lot of context and time commitment (_it's certainly difficult for me_). This segment could be a middle ground.

Reply:
@aemengo this is great feedback. Would you want this on a weekly cadence? Monthly? Quarterly?

Reply:
Monthly feels appropriate.

Reply:
IIUC "roadmap review" will be a quarterly recurring item for the Core Team Sub Sync (on Wednesdays).